### PR TITLE
#407 Improve error logging for hmac validations

### PR DESCRIPTION
**Pull request by lojzatran**
Fixes: https://github.com/commercetools/commercetools-adyen-integration/issues/407

### CODE REVIEW

File: notification/src/utils/hmacValidator.js [hunk 1]
```diff
@@ -18,10 +18,19 @@ function hasValidHmacSignature (notification) {
   }
   */
   const notificationRequestItem = notification.NotificationRequestItem
-  if (!notificationRequestItem.additionalData || !notificationRequestItem.additionalData.hmacSignature)
-    return false
-
-  return validator.validateHMAC(notificationRequestItem, config.adyen.secretHmacKey)
+  if (!notificationRequestItem.additionalData)
+    return 'Notification does not contain the required field '
```
Comment (ahmetoz) [inline notification/src/utils/hmacValidator.js:22]: I am not sure if this field is required.
https://docs.adyen.com/development-resources/webhooks/notifications-api#notificationrequestitem

Comment (ahmetoz) [reply notification/src/utils/hmacValidator.js:22]: I also could not see a test case for that?

**Comment by ahmetoz** [comment]
Also, there are some eslint issues in the notification module.

File: notification/src/utils/hmacValidator.js (hunk 1 above)
Comment (lojzatran) [reply notification/src/utils/hmacValidator.js:22]: https://github.com/commercetools/commercetools-adyen-integration/pull/465/commits/2dcac27a0eb89adfdc404c0d99be0ed1959c7195

Comment (lojzatran) [reply notification/src/utils/hmacValidator.js:22]: It's required if you enable HMAC. Check where it is being called: https://github.com/commercetools/commercetools-adyen-integration/pull/465/files#diff-03c88d579df3d6ccea32a895f34ddf21ac83c4b6a0d8809be06d1902df7e76c3R18

**Comment by lojzatran** [comment]
> Also, there are some eslint issues in the notification module.

https://github.com/commercetools/commercetools-adyen-integration/pull/465/commits/f26b874a82309af415b46eafe90369bf5df98441

**Review by ahmetoz — APPROVED** [review]

**Pull request merged**